src/snippets/get-started/node-js/Step4.mdx

```diff
@@ -4,24 +4,22 @@ import { Aside } from "@astrojs/starlight/components";
 ## 4. Start app
 
 {/* prettier-ignore */}
-<SelectableContent client:load syncKey="language" frameworkSwitcher>
-  <div slot="TS" slotIdx="1">
-  ```sh
-  npx tsx --env-file .env.local index.ts
-  ```
-  </div>
-  <div slot="JS" slotIdx="2">
-  <Aside type="note" title="env files">
+<Aside type="note" title="env files">
```

Comment:
I think we might want this notice underneath the command, as most people will be on the latest LTS node.